core/monitor/monitor_test.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package monitor
+
+import (
+	"context"
+	"testing"
+
+	pb "github.com/google/keytransparency/core/api/v1/keytransparency_proto"
+	tpb "github.com/google/trillian"
+)
+
+func TestEpochPairs(t *testing.T) {
+	ctx := context.Background()
+	for _, tc := range []struct {
+		in  []int64
+		out []struct {
+			a, b int64
+		}
+	}{
+		{in: []int64{0, 1, 2}, out: []struct{ a, b int64 }{{0, 1}, {1, 2}}},
+	} {
+		t.Log("begin")
```

Comment:
remove the t.Log() / t.Logf() calls if you're happy everything works.

Reply:
Done